rq_dashboard/web.py

```diff
@@ -160,27 +142,44 @@ def pagination_window(total_items, cur_page, per_page=5, window_size=10):
     return result
 
 
-@blueprint.route('/', defaults={'queue_name': None, 'page': '1'})
-@blueprint.route('/<queue_name>', defaults={'page': '1'})
-@blueprint.route('/<queue_name>/<page>')
-def overview(queue_name, page):
-    if queue_name == 'failed':
-        queue = get_failed_queue()
-    elif queue_name is None:
-        # Show the failed queue by default if it contains any jobs
-        failed = get_failed_queue()
-        if not failed.is_empty():
-            queue = failed
-        else:
-            queue = Queue()
+def get_queue_registry_jobs_count(queue_name, registry_name, offset, per_page):
+    queue = Queue(queue_name)
+    if registry_name == 'failed':
+        current_queue = FailedJobRegistry(queue_name)
+    elif registry_name == 'deferred':
+        current_queue = DeferredJobRegistry(queue_name)
+    elif registry_name == 'started':
+        current_queue = StartedJobRegistry(queue_name)
+    elif registry_name == 'finished':
+        current_queue = FinishedJobRegistry(queue_name)
+    else:
+        current_queue = queue
+    total_items = current_queue.count
+
+    job_ids = current_queue.get_job_ids(offset, per_page)
+    current_queue_jobs = [queue.fetch_job(job_id) for job_id in job_ids]
+    jobs = [serialize_job(job) for job in current_queue_jobs]
+
+    return(total_items, jobs)
+
+
+@blueprint.route('/', defaults={'queue_name': None, 'registry_name': 'queued', 'page': '1'})
+@blueprint.route('/<queue_name>', defaults={'registry_name': 'queued', 'page': '1'})
```

Comment:
trailing slash please

Reply:
Also, It might make sense to use something like `/queues/<queue_name>/...`, but let's do this separately.